third_party/llvm-project/compiler-rt/METADATA

```diff
@@ -0,0 +1,21 @@
+name: "compiler-rt"
+description:
+  "Filtered subtree at third_party/llvm-project/compiler-rt."
+
+third_party {
+  url {
+    type: LOCAL_SOURCE
+    value: "https://cobalt.googlesource.com/third_party/llvm-project/compiler-rt_filtered_mirror"
```

Comment:
I think one thing that we don't have in traceable form is how were the filters created. We should have at least bug logged where the filter-repo sequence with upstream and filter config was posted.

Ideally we should also set up continuous Copybara jobs with an identical subdirectory config, so that we can seamlessly rebase from there when we needed.

Reply:
updated!